### PR TITLE
fix: add Skip Link to Consent view only

### PR DESCRIPTION
**Pull request by sherizada-okta**
## Description:
Show **Skip to main content** link for Consent view only

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
Demo: https://okta.box.com/s/ru2trrv0d9d09jzbyvfxxf1tzlnqlpel

Screen Shot 2020-12-01 at 12 28 51 PM

### Issue:

- [OKTA-349886](https://oktainc.atlassian.net/browse/OKTA-349886)

### CODE REVIEW

**Comment by sherizada-okta** [comment]
Running this change against monolith https://bacon-go.aue1e.saasure.net/#!/commits/monolith/sha/c2c1a3c03b03b38fdb6a7daede3d8dab3f02124f

**Comment by sherizada-okta** [comment]
Monolith CI passes https://bacon-go.aue1e.saasure.net/#!/commits/monolith/sha/c2c1a3c03b03b38fdb6a7daede3d8dab3f02124f

File: src/ConsentRequiredController.js [hunk 1]
```diff
@@ -24,6 +25,10 @@ export default FormController.extend({
     this.model.set('expiresAt', this.options.appState.get('expiresAt'));
     this.model.set('scopes', this.options.appState.get('scopes'));
     this.listenTo(this.form, 'cancel', _.bind(this.model.cancel, this.model));
+
+    // add Skip to main content link
+    const skipLink = new SkipLink();
+    $('body').find('#okta-login-container').prepend(skipLink.render().$el);
```
Comment (haishengwu-okta) [inline src/ConsentRequiredController.js:31]:
```
$(#${Enums.WIDGET_CONTAINER_ID}).find(..).prepend(..)
```

**Review by aarongranick-okta — APPROVED** [review]

File: src/ConsentRequiredController.js (hunk 1 above)
Comment (sherizada-okta) [reply src/ConsentRequiredController.js:31]: Ok, what's the rule for when to add a class/id to the Enums file?

Comment (sherizada-okta) [reply src/ConsentRequiredController.js]: I added a new one: https://github.com/okta/okta-signin-widget/pull/1540/commits/e50036e8415fd0c72e42204680f3fbcf4f4deaff#diff-6cd1196e6f18f0fef5f0cb85e1d32142bfa44deacee6ba54a3d26d1521b06068R16

File: src/v2/BaseLoginRouter.js [hunk 2]
```diff
@@ -80,9 +79,8 @@ export default Router.extend({
     this.appState = new AppState();
 
     const wrapper = new AuthContainer({ appState: this.appState });
-    const skipLink = new SkipLink();
 
-    $(options.el).append([skipLink.render().$el, wrapper.render().$el]);
+    $(options.el).append(wrapper.render().$el);
```
Comment (pradeepdewda-okta) [inline src/v2/BaseLoginRouter.js:83]: Can you add the screenshot for V2 login-page? I don't think it's required in V2 yet, we don't have consent for v2 yet.. i guess. @haishengwu-okta  ??

Comment (pradeepdewda-okta) [reply src/v2/BaseLoginRouter.js:83]: ah.. is't just the previous code removal ?

Comment (sherizada-okta) [reply src/v2/BaseLoginRouter.js:83]: Yes just removing the code I added.

**Review by pradeepdewda-okta — APPROVED** [review]

**Review by haishengwu-okta — APPROVED** [review]

File: src/v2/BaseLoginRouter.js (hunk 2 above)
Comment (haishengwu-okta) [reply src/v2/BaseLoginRouter.js:83]: Correct, no consent in v2 yet.

**Pull request closed without merging**